### DISCUSSION TITLE
SP-API How to get FBA Minimum Level and Estimate Excess?

### DISCUSSION THREAD

**Discussion by tieguys** [amzn/selling-partner-api-models#3809] [General] [answered]
I've been digging around and I can't seem to find these numbers anywhere...?  It would make sense if they were included in a /fba/inventory/v1/summaries?details=true call, alas they are not.  

Has anyone found this data anywhere in the API?

Cheers....

**Comment by kevnhowe** [comment, marked as answer]
Look here: https://developer-docs.amazon.com/sp-api/docs/report-type-values-fba#fba-manage-inventory-health-report
This report should have all the data you are looking for.

**Reply by Waqas770** [reply]
Is there any report which give me Shipment fees Placement Fee and total weight of Shipment in one report ?
